frameworklauncher/README.md

```diff
@@ -26,7 +26,7 @@ FrameworkLauncher (or Launcher for short) is built to enable running Large-Scale
 * **High Availability**
   * All Launcher and Hadoop components are Recoverable and Work Preserving. So, User Services is by designed No Down Time, i.e. always uninterrupted when our components shutdown, crash, upgrade, or even any kinds of outage for a long time.
```

Comment:
>Recoverable [](start = 43, length = 11)

recoverable ? shouldn't upper cases ?

Reply:
This is by design as aligned with Work Preserving

---
In reply to: [199691277](https://github.com/Microsoft/pai/pull/798#discussion_r199691277) [](ancestors = 199691277)